presto-main/src/main/java/io/prestosql/sql/analyzer/FeaturesConfig.java

```diff
@@ -965,6 +966,18 @@ public FeaturesConfig setLateMaterializationEnabled(boolean lateMaterializationE
         return this;
     }
 
+    public boolean isDeprecatedFunctionWarningEnabled()
+    {
+        return deprecatedFunctionWarningEnabled;
+    }
+
+    @Config("analyzer.deprecated-function-warning-enabled")
+    public FeaturesConfig setDeprecatedFunctionWarningEnabled(boolean deprecatedFunctionWarningEnabled)
+    {
+        this.deprecatedFunctionWarningEnabled = deprecatedFunctionWarningEnabled;
+        return this;
+    }
+
```

Comment:
Same here... I don't think this is necessary. This change does not affect behavior or break compatibility, so there's no need to make it configurable.